tests/containers/push_container_image_to_acr.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Push a container image to the Azure Cloud Registry
+#
+# Maintainer: Ivan Lausuch <ilausuch@suse.com>, qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use containers::urls 'get_suse_container_urls';
+
+sub run {
+    my ($self, $args) = @_;
+
+    $self->select_serial_terminal;
+
+    my $provider = $self->provider_factory(service => 'ACR');
+    my ($untested_images, $released_images) = get_suse_container_urls();
+    my $image = $untested_images->[0];
+    my $tag = $provider->get_default_tag();
+
+    record_info('Pull', "Pulling $image");
+    assert_script_run("docker pull $image", 360);
+
+    my $image_build_format = '{{ index .Config.Labels "org.opencontainers.image.version"}}';
```

Comment:
not sure but do you need an space at the end? `'{{ index .Config.Labels "org.opencontainers.image.version" }}'`

Reply:
I is not required, but I added anyway to increase the readability https://docs.docker.com/config/formatting/